### ISSUE TITLE
quickscope and GTK3 warnings stop it from running

### ISSUE THREAD

**Issue by lanceman2** [lanceman2/quickscope#1]
quickscope is getting the GTK warning:
(quickplot:26655): WARNING **: Couldn't connect to accessibility bus: Failed to connect to socket /tmp/dbus-6zgWFqbtLW: Connection refused

setting env NO_AT_BRIDGE=1 lets it continue running.  But maybe we should changing the GTK setting the the gtk init stuff so that it does not fail on GTK warnings.
